userspace/libsinsp/parsers.cpp

```diff
@@ -2296,6 +2309,12 @@ void sinsp_parser::parse_open_openat_creat_exit(sinsp_evt *evt)
 			parinfo = evt->get_param(5);
 			ASSERT(parinfo->m_len == sizeof(uint32_t));
 			dev = *(uint32_t *)parinfo->m_val;
+			if (evt->get_num_params() > 6)
+			{
+				parinfo = evt->get_param(6);
```

Comment:
Does `evt->get_param(6)` always return non-NULL here? If not, check for `nullptr` before dereferencing `parinfo`.

Reply:
AFAICT from reading `get_param` code it is always non-NULL. Also, this access pattern (`get_param`, then immediately dereference) is all over the place in this file, including the three lines preceding my change

Reply:
Agreed, it's better to be consistent.